### PR TITLE
CreateDropdown: change navigation to specific image <HMS-4258>

### PR DESCRIPTION
**Pull request by mgold1234**
this commit change navigation to specific image when creating and building new image

### CODE REVIEW

**Comment by regexowl** [comment]
@mgold1234 Can you please resolve the lint errors?

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

File: src/Components/CreateImageWizardV2/steps/Review/Footer/CreateDropdown.tsx [hunk 1]
```diff
@@ -41,7 +43,19 @@ export const CreateSaveAndBuildBtn = ({
...
-    blueprint && buildBlueprint({ id: blueprint.id, body: {} });
+
+    if (blueprint) {
+      buildBlueprint({ id: blueprint.id, body: {} });
+      dispatch(setBlueprintId(blueprint.id));
+
+      setTimeout(() => {
+        const row = document.getElementById(`blueprint-row-${blueprint.id}`);
+        if (row) {
+          row.click();
```
Comment (ezr-ondrej) [inline src/Components/CreateImageWizardV2/steps/Review/Footer/CreateDropdown.tsx:54]: I wonder why do we need to click?
In my mind we should: invalidate the Blueprints data so we refetch and then dispatch the blueprint ID to select the blueprint, right?

Or is the click needed for some other reasons?

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

**Review by ezr-ondrej — DISMISSED** [review]
Looks sane to me :)
Do you think we should add tests, or would it be disproportionatelly difficult to the simplicity of the patch?
@regexowl ? :)

**Comment by regexowl** [comment]
Adding tests might be in this case far more complex than the patch so I'm perfectly fine with testing this one manually. Also this is a functionality that should trigger on every blueprint creation so it not working should be easy to catch.

That being said - when I create a blueprint it doesn't get automatically selected for me:

create-blueprint

**Comment by mgold1234** [comment]
@regexowl I understand from the task itslef that it is only when user build an image, sohuld we add it in the creation as well?

**Comment by regexowl** [comment]
I didn't finish my teapot of green tea yet :sweat_smile: But when you are building an image you already have a blueprint selected, right? Since the "Build images" button is visible only when `selectedBlueprintId !== undefined`

From what I understand looking at the task - when you create a blueprint, you get currently sent to "All images". Instead of this we want to send user directly to the "blueprint page" (automatically select the blueprint in the side bar) they just created so they don't need to click on it again to build images.

**Comment by mgold1234** [comment]
that's make sense, thank you

**Comment by regexowl** [comment]
> @regexowl the selection only happens when you "Create and build" Although I do not understand why 😛
> 
> Unless it was a request from someone, I'd select the blueprint even if we're just Creating the blueprint :)

I see! Overlooked that :sweat_smile: Okay, yeah then I think we need it for "Create" also.

**Comment by ezr-ondrej** [comment]
Looks good, but the linter is unhappy :)

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

**Review by ezr-ondrej — APPROVED** [review]
Nice! Works as expected! :+1:

**Pull request merged**